vault/decrypt/decrypt.go

```diff
@@ -0,0 +1,85 @@
+package decrypt
+
+import (
+	"encoding/base64"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	vaultclient "github.com/hashicorp/vault/api"
+)
+
+// Config represents the configuration used to create a new vault decrypting
+// value modifier.
+type Config struct {
+	VaultClient *vaultclient.Client
+	KeyRingName string
+}
+
+// DefaultConfig provides a default configuration to create a new vault
+// decrypting value modifier by best effort.
+func DefaultConfig() Config {
```

Comment:
Can be droppped as AFAIK we don't use defaults in this way but init values directly via `&Config{}`